extension/types/file-size.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'filesize' {
+  export function filesize(
+    bytes: number,
+  ): string;
+}
```

Comment:
there are built-in types in `filesize.js` (https://github.com/avoidwork/filesize.js/blob/master/types/filesize.d.ts#L55), probably we can use them?